YAML files reference docs: labels and users coming soon, ticket creation, ABM and VPP (#24480)
fleetdm/fleet: docs/Configuration/yaml-files.md

The `software` section allows you to configure packages and Apple App Store apps that you want to install on your hosts.

Currently, managing [Fleet-maintained apps](https://fleetdm.com/guides/install-fleet-maintained-apps-on-macos-hosts) is only supported using Fleet's UI or [API](https://fleetdm.com/docs/rest-api/rest-api) (YAML coming soon).

- `packages` is a list of paths to custom packages (.pkg, .msi, .exe, .rpm, or .deb).
- `app_store_apps` is a list of Apple App Store apps.

> Currently, all supported platforms for App Store apps are added. For example, when you add [Bear](https://apps.apple.com/us/app/bear-markdown-notes/id1016366447) (supported on iOS and iPadOS) both the iOS and iPadOS apps will be added to your software that's available to install in Fleet. Specifying specific platforms is only supported using Fleet's UI or [API](https://fleetdm.com/docs/rest-api/rest-api) (YAML coming soon).

Author: ```suggestion
Currently, one app for each of an App Store app's supported platforms are added. For example, adding [Bear](https://apps.apple.com/us/app/bear-markdown-notes/id1016366447) (supported on iOS and iPadOS) adds both the iOS and iPadOS apps to your software that's available to install in Fleet. Specifying specific platforms is only supported using Fleet's UI or [API](https://fleetdm.com/docs/rest-api/rest-api) (YAML coming soon).
```

### integrations

> The `integrations` section lets you configure your Google Calendar, Jira, and Zendesk. After configuration, you can enable automations like calendar event and ticket creation for failing policies and vulnerabilities. Currently, enabling automation is only available using Fleet's UI or [API](https://fleetdm.com/docs/rest-api/rest-api) (YAML files coming soon). Learn more about automations in Fleet [here](https://fleetdm.com/docs/using-fleet/automations).

Author: ```suggestion
The `integrations` section lets you configure your Google Calendar, Jira, and Zendesk. After configuration, you can enable automations like calendar event and ticket creation for failing policies and vulnerabilities. Currently, enabling automations is only available using Fleet's UI or [API](https://fleetdm.com/docs/rest-api/rest-api) (YAML files coming soon). Learn more about automations in Fleet [here](https://fleetdm.com/docs/using-fleet/automations).
```

> The `integrations` section lets you configure your Google Calendar, Jira, and Zendesk. After configuration, you can enable automations like calendar event and ticket creation for failing policies and vulnerabilities. Currently, enabling automations is only available using Fleet's UI or [API](https://fleetdm.com/docs/rest-api/rest-api) (YAML files coming soon). Learn more about automations in Fleet [here](https://fleetdm.com/docs/using-fleet/automations).

Reviewer: You can enable some automations via YAML ([example](https://fleetdm.com/docs/configuration/yaml-files#separate-file)). Or is this about a different kind of automation? (If so, I think we need to make that language clearer.)

Author: Ah, good catch. I think it's specifically ticket creation that you can't enable via YAML. I updated this section to the following:

_The `integrations` section lets you configure your Google Calendar, Jira, and Zendesk. After configuration, you can enable automations like calendar event and ticket creation for failing policies and vulnerabilities. Learn more about automations in Fleet [here](https://fleetdm.com/docs/using-fleet/automations).

Currently, enabling ticket creation is only available using Fleet's UI or [API](https://fleetdm.com/docs/rest-api/rest-api) (YAML files coming soon)._
